### ISSUE TITLE
Map, inventory, skills etc are all missing information

### ISSUE THREAD

**Issue by Riemu-Makkonen** [iLPdev/prs#150]
Hello,
I'm using Mudlet 4.18.5, Windows11 and just installed 1.8.0 package, all other windows beside the chat windows are not working

![image](https://github.com/user-attachments/assets/2422b33e-f7ee-4495-9102-084db56b4bf9)

**Comment by iLPdev** [comment]
Yes, the game server changed the GMCP it sends and no one has ever updated this script. Specifically, `prs-battle` needs to be modified to accept the new formatted GMCP data. I've no time to look into it further but someone who knows what they're doing could easily fix this and then I could push out a fix version of the package.

**Comment by Riemu-Makkonen** [comment]
These are the scripts I made with Chatgpt to get the hp/food etc. gauges and minimap working allthough the rage one might not work at the moment, these might be of help to real coders. gmcp.Char.room doesn't exist anymore > chatgpt changed it to gmcp.room.info

[prs-mapper.txt](https://github.com/user-attachments/files/16715106/prs-mapper.txt)
[PRS-gauges.txt](https://github.com/user-attachments/files/16715109/PRS-gauges.txt)
